rest_api/limsdb/queries/sample_status.py

```diff
@@ -279,13 +279,13 @@ def _create_samples(session):
     return all_samples.values()
 
 
-def sample_status(session):
+def sample_status(session, app):
```

Comment:
Any reason we're adding `app` to the args in these functions? Is this a work in progress?

Edit: I see that this is needed in `run_status` - see comment there